mlflow/go/pkg/tracking/service/experiments.go

```diff
@@ -68,13 +68,28 @@ func (m TrackingService) GetExperiment(input *protos.GetExperiment) (*protos.Get
 	return &response, nil
 }
 
-func (m TrackingService) DeleteExperiment(
+func (ts TrackingService) DeleteExperiment(
 	input *protos.DeleteExperiment,
 ) (*protos.DeleteExperiment_Response, *contract.Error) {
-	err := m.Store.DeleteExperiment(input.GetExperimentId())
+	err := ts.Store.DeleteExperiment(input.GetExperimentId())
 	if err != nil {
 		return nil, err
 	}
 
 	return &protos.DeleteExperiment_Response{}, nil
 }
+
+func (ts TrackingService) GetExperimentByName(
+	input *protos.GetExperimentByName,
+) (*protos.GetExperimentByName_Response, *contract.Error) {
+	experiment, cErr := ts.Store.GetExperimentByName(input.GetExperimentName())
+	if cErr != nil {
```

Comment:
if we name this `cErr`, then we probably want to name it like that everywhere in this file